server/src/test/java/io/crate/integrationtests/RenameTableIntegrationTest.java

```diff
@@ -145,6 +157,55 @@ public void testRenamePartitionedTablePartitionStayClosed() {
         execute("alter table tp1 rename to tp2");
 
         execute("select closed from information_schema.table_partitions where partition_ident = '04132'");
-        assertThat(response.rows()[0][0], is(true));
+        assertThat((boolean) response.rows()[0][0]).isTrue();
+    }
+
+    @Test
+    public void test_cannot_rename_table_if_target_already_exists() {
+        String schema = sqlExecutor.getCurrentSchema();
+
+        execute("create table t1(a int)");
+        execute("create table t2(b int)");
+        execute("create table t1_parted(a int) partitioned by(a)");
+        execute("create table t2_parted(b int) partitioned by(b)");
+        Asserts.assertSQLError(() -> execute("alter table t1 rename to t2"))
+            .hasPGError(PGErrorStatus.INTERNAL_ERROR)
+            .hasHTTPError(HttpResponseStatus.BAD_REQUEST, 4000)
+            .hasMessageContaining(String.format(
+                Locale.ENGLISH,
+                "Cannot rename table %s.t1 to %s.t2, table %s.t2 already exists",
```

Comment:
I feel like for parted tables only partitioned target is interesting since in this PR we are doing lookups only for targets. 

Maybe we can drop 
`alter table t1_parted rename to v1` and `alter table t1_parted rename to t2` scenarios?

But if you think it's needed feel free to keep it  - I just can't think of scenarios where partitioned source can be affected by this PR

Reply:
I agree, my thinking is only that maybe in the future there is some refactoring on the code in the transport action and the executor, .... and I'm scared something could break, on the other hand I think I'm overdoing it, so I'll remove. thx!